pkg/network/network.go

```diff
@@ -265,6 +265,9 @@ func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
 	return nc, nil
 }
 
+// GetDomainTemplate returns the golang Template from the config map.
```

Comment:
```suggestion
// GetDomainTemplate returns the golang Template from the config map
```

You already have a period at the end